### PR TITLE
Adding Azure Meetup Cologne

### PR DESCRIPTION
**Pull request by christianwaha**

### CODE REVIEW

**Review by davidjrh — CHANGES_REQUESTED** [review]
@christianwaha the current Sessionize endpoint is not returning JSON. Please, double check the Sessionize endpoint settings available at https://blog.globalazure.net/Blog/Post/284/Sessionize-integration-setup-for-local-event-organizers-for-2022-edition, ensuring that returns JSON, each session has valid startsAt and endsAt values and that the fields mentioned in the post are available.

https://sessionize.com/api/v2/xrmtfv0v/view/Sessions

File: 2022/azurecgn/data.json [hunk 1]
```diff
@@ -9,7 +9,7 @@
     "Name": "Azure Meetup Cologne",
     "Country": "Germany",
     "Url": "https://www.rakoellner.de/global-azure-cgn/",
-    "SessionizeApiId": "xrmtfv0v",
```
Comment (noopman) [inline 2022/azurecgn/data.json]: Hello @christianwaha ! When you browse to this url with your ApiId it should return valid json. There is something not correctly configured in your Sessionize! Please take a look at that and resolve! /Magnus

**Review by noopman — CHANGES_REQUESTED** [review]

File: 2022/azurecgn/data.json (hunk 1 above)
Comment (christianwaha) [reply 2022/azurecgn/data.json]: Hi Magnus I thought I pushed a version without the API key, maybe you can remove it and push it to be visible on the website?

**Comment by christianwaha** [comment]
@noopman The newest version i pushed some minutes ago is without the sessionize key.

**Review by noopman — CHANGES_REQUESTED** [review]
Now all I am missing is an index.md file! Will you add one @christianwaha, please!

**Comment by christianwaha** [comment]
@noopman I added an index.md.

**Pull request merged**